faq.md

```diff
@@ -47,6 +59,14 @@ You will find the system name in the url of the monitor in Sigrid. The structure
 
 Please send and email to support@softwareimprovementgroup.com and mention your name, your company and the system name(s) that you need a token for. We will send the account via mail and the token via sms.
 
+### What target quality should we use?
+
+Avoid setting an unreasonably high target quality level. While it seems appealing to be ambitious, this can be demotivating or frustrating to people, as it's not always possible to achieve such a high level in every single change. . 
```

Comment:
```suggestion
Avoid setting an unreasonably high target quality level. While it seems appealing to be ambitious, this can be demotivating or frustrating to people, as it's not always possible to achieve such a high level in every single change.
```